combat-trainer.lic

```diff
@@ -2019,6 +2019,7 @@ class AbilityProcess
       end
     @barb_buffs
       .select { |name| Flags["ap-#{name}-expired"] }
+      .reject { |name| DRSpells.active_spells[name] }
```

Comment:
Although we're updating `DRCA.activate_barb_buff?` to have this same check, doing the check here in `combat-trainer` also means we can avoid the inner fire warning message that happens a couple lines below if there really isn't any abilities that need to be restarted.